vagrant/base.sh

```diff
@@ -31,9 +31,9 @@ if [ -z `which javac` ]; then
     if [ -e "/tmp/oracle-jdk8-installer-cache/" ]; then
         find /tmp/oracle-jdk8-installer-cache/ -not -empty -exec cp '{}' /var/cache/oracle-jdk8-installer/ \;
     fi
-    if [ ! -e "/var/cache/oracle-jdk8-installer/jdk-8u171-linux-x64.tar.gz" ]; then
+    if [ ! -e "/var/cache/oracle-jdk8-installer/jdk-8u191-linux-x64.tar.gz" ]; then
```

Comment:
this seems unrelated to this PR? and the bucket probably shouldn't change unless we're changing the target bucket -- we want only 1, permanent bucket if possible